Updated stuck_path, minor details

LeonardWarrior opened the pull request:
Changed the timings to not confure seconds with milliseconds.

Made it go through more loops of unstick, which progressively should make it get out of most sticky situations. Added so that it correctly walks backward, didn't work for me in original code since for some reason it was pressing arrow forward nonstop

julianperrott commented on Libs/Path/StuckDetector.cs:
```diff
@@ -84,35 +84,36 @@ public async Task Unstick()
                 {
                     logger.LogInformation($"Trying to unstick by strafing left for {strafeDuration}ms");
-                    wowProcess.SetKeyState(ConsoleKey.Q, true, false, "StuckDetector");
+                    wowProcess.SetKeyState(ConsoleKey.A, true, false, "StuckDetector");
```
This isn't strafing anymore it is turning.

LeonardWarrior replied:
Ah this change is not what I wanted to highlight hehe, it's just because I have strafe on those. Ignore this

julianperrott commented:
My unsticking code needs rewriting as it isn't very good. Jumping seems to solve most problems, so it has been a bit neglected.

LeonardWarrior commented:
The only change that is really needed is the part where I changed the timings, strafeduration for example is used in await Task.Delay(strafeDuration); which is in milliseconds.

The other part is where I added wowProcess.SetKeyState(ConsoleKey.UpArrow, false, false, "StuckDetector"); since otherwise he would not back up. With this it becomes very robust, and he finds his way out of almost any stuck place.
